File: java/Warmup-2/stringMatch.java
public int stringMatch(String a, String b) {
  return a.length() < 2 || b.length() < 2 ? 0 : (a.substring(0, 2).equals(b.substring(0, 2)) ? 1 : 0) + stringMatch(a.substring(1), b.substring(1));
}

File: python/Warmup-2/string_match.py
string_match = lambda a, b : len([1 for i in range(len(b) - 1) if a[i : i + 2] == b[i : i + 2]]) if len(a) > len(b) else len([1 for i in range(len(a) - 1) if b[i : i + 2] == a[i : i + 2]])
